src/libraries/System.Private.CoreLib/src/System/Char.cs

```diff
@@ -384,7 +384,7 @@ public static char ToLower(char c)
         // Converts a character to lower-case for invariant culture.
         public static char ToLowerInvariant(char c)
         {
-            return CultureInfo.InvariantCulture.TextInfo.ToLower(c);
+            return TextInfo.Invariant.ToLower(c);
```

Comment:
I'd assumed I guess incorrectly that devirtualization was kicking in and enabling this to be direct calls... but I guess not?